cluster/gce/gci/configure-helper.sh

```diff
@@ -364,6 +364,13 @@ function ensure-local-ssds() {
   # The following mounts or symlinks NVMe devices
   get-local-disk-num "nvme" "block"
   local nvmeblocknum="${localdisknum}"
+  get-local-disk-num "nvme" "fs"
+  local nvmefsnum="${localdisknum}"
+  # Check if NVMe SSD specified.
+  if [ "${nvmeblocknum}" -eq "0" ] && [ "${nvmefsnum}" -eq "0" ]; then
+    echo "No local NVMe SSD specified."
```

Comment:
Is it more accurate to say here that an NVMe SSD boot disk has been detected? Because a blocknum of 0 doesn't mean there is no NVMe SSD IIUC.

If the boot disk is SCSI, could there be an NVMe SSD with blocknum 0?

Reply:
`nvmeblocknum` refers to how many nvme raw block local ssds were requested in NODE_LOCAL_SSDS_EXT, and `nvmefsnum` refers to how many nvme formatted local ssds were requested.

Reply:
Ah, I see now.  Thanks

LGTM